eemeter/metrics.py

```diff
@@ -310,8 +310,11 @@ def __init__(
         self.num_parameters = num_parameters
         self.autocorr_lags = autocorr_lags
 
-        self.observed_mean = combined["observed"].mean()
-        self.predicted_mean = combined["predicted"].mean()
+        # to account for solar usage the cvrmse should be calculated as the
+        # mean of the absolute value of observed.
+
+        self.observed_mean = abs(combined["observed"]).mean()
+        self.predicted_mean = abs(combined["predicted"]).mean()
```

Comment:
This looks funny to me `self.observed_mean = abs(combined["observed"]).mean()` - should that be `self.observed_mean = abs(combined["observed"].mean())` ?

Are you intending to take the absolute value of the mean or of all the values going into the mean? If the former (and that seems less correct to me anyway) it might be more idiomatic to use `.abs()`

Reply:
The idea is to take the absolute value of all values going into the mean. Solar data has negative values, so the mean(obs) can be very close to zero. Taking the mean(abs(obs)) gives a more appropriate estimate of the magnitude of observed values, making cvrmse interpretable. I am open to changing how it is written.

Reply:
Gotcha!